src/widgets/radio.jsx

```diff
@@ -19,6 +19,146 @@ var captureScratchpadTouchStart =
 
 var cx = React.addons.classSet;
 
+var Choice = React.createClass({
+    propTypes: {
+        checked: React.PropTypes.bool,
+        classSet: React.PropTypes.shape,
+        clue: React.PropTypes.shape,
+        correct: React.PropTypes.bool,
+        content: React.PropTypes.shape,
+        disabled: React.PropTypes.bool,
+        groupName: React.PropTypes.string,
+        onChecked: React.PropTypes.func,
+        type: React.PropTypes.string
+    },
+
+    getDefaultProps: function() {
+        return {
+            checked: false,
+            classSet: {},
+            correct: false,
+            disabled: false,
+            type: 'radio'
+        };
+    },
+
+    render: function() {
+        this.props.classSet[ApiClassNames.RADIO.OPTION] = true;
+        this.props.classSet[ApiClassNames.RADIO.SELECTED] = this.props.checked;
+
+        return <div className={cx(this.props.classSet)}>
+            <span className="checkbox">
+                <input
+                    type={this.props.type}
+                    name={this.props.groupName}
+                    checked={this.props.checked}
+                    disabled={this.props.disabled}
+                    onClick={(e) => {
+                        // Avoid sending this to the parent
+                        e.stopPropagation();
+                    }}
+                    onChange={(e) => {
+                        this.props.onChecked(e.target.checked);
+                    }} />
+            </span>
+            {/* A pseudo-label. <label> is slightly broken on iOS,
+                so this works around that. Unfortunately, it is
+                simplest to just work around that everywhere. */}
+            <span className={
+                    ApiClassNames.RADIO.OPTION_CONTENT + " " +
+                    ApiClassNames.INTERACTIVE
+                }
+                style={{ cursor: "default" }}>
+                <div>
+                    {this.props.content}
+                </div>
+            </span>
+            {Exercises.cluesEnabled && this.props.checked &&
+                <div className="perseus-radio-clue">
+                    {this.props.clue}
+                </div>}
+        </div>
+    }
+});
+
+var ChoiceNoneAbove = React.createClass({
+    getDefaultProps: function() {
+        return {
+            correct: true
+        }
+    },
+
+    render: function() {
+        if (!this.props.correct) {
```

Comment:
Yeah this thing is the one I'm the least satisfied with, and I still haven't made up my mind yet on how to improve it. 

The none-of-above choice needs to hide its content until it's been marked as correct (that is, when the question is answered correctly), and this is what this component does in its `render` method. But this component is also used in `RadioEditor` where the choice's content (that is, the editor) needs to be visible always.

So the idea was to have the `correct` prop of `ChoiceNoneAbove` to start off as `true` (for the editor), and then have the `Radio` component set the actual value using the `this.props.questionCompleted && values[i]` statement.

The obvious problem with this is that it's pretty cryptic. I thought of adding a comment to explain the intention behind it, but if you need a comment to explain what you're doing, then maybe your code needs some refactoring : )

A different approach would be to remove the `getDefaultProps` altogether, and just have `RadioEditor` marks all the choices as with `correct` as `true`, so that the editor is visible:

```javascript
    showDelete={this.props.choices.length >= 2} />,
isNoneOfTheAbove: choice.isNoneOfTheAbove,
checked: choice.correct
correct: choice.correct //Always show the editor
``` 

i think this one could be better, but still a bit confusing. Maybe it's just down to having a better name for the props, like instead of `correct` something like `showContent`, `reveal`, or something like that. Sometimes a better named var just makes the difference

I'm open to feedback on the matter!